server/rpc/concept/TypeHandler.java

```diff
@@ -207,9 +207,13 @@ private void isAbstract(Transaction.Req request, Type type) {
     }
 
     private void getSupertype(Transaction.Req request, Type type) {
+        ConceptProto.Type.GetSupertype.Res.Builder getSupertypeRes = ConceptProto.Type.GetSupertype.Res.newBuilder();
+        Type superType = type.getSupertype();
+        if (superType != null) {
```

Comment:
`superType` can be `null` when `type` is `thing`. So we need to do a null check here before adding to response.